pinot-api/src/main/java/org/apache/pinot/client/JsonAsyncHttpPinotClientTransport.java

```diff
@@ -58,7 +65,13 @@ public BrokerResponse executeQuery(String brokerAddress, String query)
 
       final String url = "http://" + brokerAddress + "/query";
 
-      final Future<Response> response = _httpClient.preparePost(url)
+      AsyncHttpClient.BoundRequestBuilder request = _httpClient.preparePost(url);
+
+      if(_headers != null) {
+        _headers.forEach( (k,v) -> request.addHeader(k, v));
```

Comment:
nit: please remove the extra space bet "(" and add a space after "k,"

Reply:
Thanks. Made the changes.